src/Build/BackEnd/Components/Logging/ProjectLoggingContext.cs

```diff
@@ -38,7 +37,8 @@ internal ProjectLoggingContext(NodeLoggingContext nodeLoggingContext, BuildReque
             requestEntry.RequestConfiguration.Project.PropertiesToBuildWith,
             requestEntry.RequestConfiguration.Project.ItemsToBuildWith,
             requestEntry.Request.ParentBuildEventContext,
-            requestEntry.RequestConfiguration.Project.EvaluationId
+            requestEntry.RequestConfiguration.Project.EvaluationId,
+            requestEntry.Request.BuildEventContext.ProjectContextId // TODO: Bad
```

Comment:
:cough:

Reply:
Woops, yea that was "between commits". The overall change should be correct though, so if we squash merge it should be fine. If you prefer to merge multiple commits I'll go back and clean it up. Let me know.